pullrequest.rst

```diff
@@ -170,6 +172,11 @@ Then push your work to your clone on GitHub::
 
 Make a pull request on GitHub from your changes in ``MY_BRANCH_NAME``.
 
+After your PR has been accepted and merged, you can :ref:`delete the branch <deleting_branches>`::
+
+   git branch -d MY_BRANCH_NAME  # delete local branch
```

Comment:
The `ALL_CAPS_VARIABLE` is inconsistent with the `<variable>` used elsewhere, but it's unrelated to this PR so I didn't change it.